supervisor/host/__init__.py

```diff
@@ -85,8 +85,11 @@ def supported_features(self) -> List[HostFeature]:
         if self.sys_dbus.timedate.is_connected:
             features.append(HostFeature.TIMEDATE)
 
+        if self.sys_dbus.agent.is_connected:
+            features.append(HostFeature.AGENT)
+
         if self.sys_hassos.available:
-            features.append(HostFeature.HASSOS)
+            features.append(HostFeature.HAOS)
```

Comment:
This will be breaking for the frontend, maybe more

Reply:
Right, but should be only frontend. I can make the naming of the string still hassos until we fixed the UI

Reply:
If it's only the UI, it's fine, we can just fix that before this is released.

Reply:
Let's kick legacy out, will finish review of this PR later today, and can prepare a frontend PR after that

Reply:
Frontend PR: <https://github.com/home-assistant/frontend/pull/9477>
Docs need update <https://developers.home-assistant.io/docs/api/supervisor/endpoints#host>

Reply:
done